### PR TITLE
[#1186] Keywords are now saved without clicking a button to save.

### PR DESCRIPTION
**Pull request by mauriel133**

### CODE REVIEW

**Comment by hydrobuild** [comment]
Can one of the admins verify this patch?

**Comment by pkdash** [comment]
@shawncrawley Can you review this?

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -1035,6 +1035,7 @@ <h3 style="color: red" id="file-type-error"></h3>
                 $($(this).closest(".tags").parent().find(".btn-primary")[0]).show();    // Show save button
```
Comment (shawncrawley) [inline hs_core/templates/pages/genericresource.html]: Shouldn't you remove this line if you are not wanting to show the save button?

Comment (mauriel133) [inline hs_core/templates/pages/genericresource.html]: Correct. Fixed.

File: hs_core/templates/pages/genericresource.html [hunk 2]
```diff
@@ -1066,6 +1066,7 @@ <h3 style="color: red" id="file-type-error"></h3>
                 }
```
Comment (shawncrawley) [inline hs_core/templates/pages/genericresource.html]: Line 1063 also appears to be code to show the save button that can be removed:
$(this).closest("form-control").find("button").show()

Comment (mauriel133) [inline hs_core/templates/pages/genericresource.html]: Fixed.

**Comment by shawncrawley** [comment]
+1

**Pull request merged**